kernel/nativelambda.ml

```diff
@@ -21,7 +21,7 @@ type lambda = Nativevalues.t Genlambda.lambda
 
 let is_value lc =
   match lc with
-  | Lval _ | Lint _ | Luint _ | Lfloat _ -> true
+  | Lval _ | Lint _ | Luint _ | Lfloat _ | Lstring _ -> true
   | _ -> false
```

Comment:
Not having a catchall here would have avoided the oversight.

Reply:
This can go in another PR as a cleanup, I think.

Reply:
Sure